src/connector.js

```diff
@@ -461,17 +475,13 @@ function initEngine() {
 					open: '<strong>',
 					close: '</strong>',
 				},
-				correctionDelimiters: {
-					open: '<em>',
-					close: '</em>',
-				},
 			},
```

Comment:
Can you please document this change in the description of your PR? Or open a separate PR with this change, since it is unrelated to QS.